lib/ava-files.js

```diff
@@ -1,9 +1,12 @@
 var fs = require('fs');
 var path = require('path');
+var nodePath = require('path');
```

Comment:
yeah - it's from combining the two codebases. `nodePath` was used in watcher, because he used a `path` variable inside methods. I can clean that up.